src/helpers/attendances.js

```diff
@@ -74,10 +74,10 @@ const formatCourseWithAttendances = (course, specificCourseTrainees, specificCou
         const isTraineeOnlySubscribedToSpecificCourse =
```

Comment:
et est ce qu'on ne pourrait pas ajouter un match sur `trainees` dans le populate sur les émargements dans la methode `listUnsubscribed` pour ne recuperer que ceux des apprenants de la formations comme ca on peut ne pas faire cette verification egalement ?

Reply:
il faut aussi spécifier qu'ils ne sont pas dans la formation qu'on est en train de récupérer pour répondre à la deuxième partie de la condition, ça ne marche pas avec $nin: this.trainees, tu sais comment l'écrire ?

Reply:
ah oui en effet on ne peut pas le mettre dans le populate directement car une partie de la condition est dans l'objet qu'on requete